docs/Changelog.md

```diff
@@ -11208,6 +11246,264 @@ This version of the operator has been available since version 11 of the default
 <dd>Constrain indices to integer types</dd>
 </dl>
 
+### <a name="SequenceAt-11"></a>**SequenceAt-11**</a>
+
+  Outputs a tensor copy from the tensor at 'position' in 'input_sequence'.
+  Accepted range for 'position' is in `[-n, n - 1]`, where `n` is the number of tensors in 'input_sequence'.
+  Negative value means counting positions from the back.
+
+#### Version
+
+This version of the operator has been available since version 11 of the default ONNX operator set.
+
+#### Inputs
+
+<dl>
+<dt><tt>input_sequence</tt> : S</dt>
+<dd>Input sequence.</dd>
+<dt><tt>position</tt> : I</dt>
```

Comment:
I think we'll want to take `start`, `end`, and `step` like `Slice` op?

Reply:
That will probably be a separate operator(or function). This operator returns `Tensor`, while the `Slice` equivalent returns `Sequence`.

Reply:
Ah yeah, that makes sense!